libpod/events/logfile.go

```diff
@@ -27,15 +27,17 @@ func (e EventLogFile) Write(ee Event) error {
 	}
 	lock.Lock()
 	defer lock.Unlock()
-	f, err := os.OpenFile(e.options.LogFilePath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0700)
+	eventJSONString, err := ee.ToJSONString()
 	if err != nil {
 		return err
 	}
-	defer f.Close()
-	eventJSONString, err := ee.ToJSONString()
+	// Rotates the log file
+	rotateLog(e.options.LogFilePath, eventJSONString, 500)
```

Comment:
We need to check the error.  The limit is currently hard-coded to `500` but we need to consult containers.conf.